types/test/sequelize.ts

```diff
@@ -49,4 +49,8 @@ Sequelize.afterConnect(() => {
 
 const rnd: Fn = sequelize.random();
 
-const myModel: typeof Model = sequelize.models.asd;
```

Comment:
I think you should keep this, just rename it so it does not conflict with your new lines below. Or this no longer works?

Reply:
Nah, it still works.  I can re-add if you like, although I was assuming that the line below would adequately cover the same use-case:

```ts
const myModel: typeof Model1 = sequelize.models.asd;
```

Reply:
That being said, it's not really a valid use-pattern, because you'd end up running into the issues that we're trying to address, because we're casting to a less-flexible type. 

```ts
const myModel: typeof Model = sequelize.models.asd;
myModel.findAll(); // <-- TS error
```

Reply:
Ah, ok :+1: so keep it removed. Thanks for the explanation!